cg/store/crud/create.py

```diff
@@ -398,12 +397,13 @@ def add_organism(
             **kwargs,
         )
 
-    def add_order(self, customer: Customer, ticket_id: int) -> Order:
+    def add_order(self, customer: Customer, ticket_id: int, **kwargs) -> Order:
```

Comment:
Why did this change?

Reply:
I made this function more similar to the others in this module (specially add_sample). This allows you to add any valid argument to the model as parameter to the function. Not currently used as we only use ticket and customer to initialise Orders, but I think this could be used in the future